src/components/KeyboardLayout/KeyboardSelect.tsx

```diff
@@ -0,0 +1,35 @@
+import Select from "react-select";
+import { KBLayout, KBLayoutType } from "../../enums/KBLayout";
+import { Dispatch, SetStateAction } from "react";
+
+type Props = {
+  setLayout: Dispatch<SetStateAction<KBLayoutType>>;
+};
+
+export const KeyboardSelect = (props: Props) => {
+  const { setLayout } = props;
+
+  const options: { value: string; label: string }[] = Object.entries(
```

Comment:
I don't like that we are recomputing this every time the component renders. Can we memoise it?